hamster_gtk/helpers.py

```diff
@@ -177,6 +180,23 @@ def decompose_raw_fact_string(text, raw=False):
     return result
 
 
+# [TODO]
+# Oncec LIB-251 has been fixed this should no longer be needed.
+def get_recent_activities(controller, start, end):
+    """Return a list of all activities logged in facts within the given timeframe."""
+    recent_activities = [fact.activity for fact in controller.facts.get_all(start=start, end=end)]
+    return OrderedSet(recent_activities)
+
+
+def serialize_activity(activity):
```

Comment:
Do not we have something like this somewhere already?

Reply:
I think earlier ``hamster-lib`` branches had something like this. Also the overview has its own "serialize activity" code which is different however:
- it deals with legacy hamerest representation of ``activity.category=None``. This is something that I would be willing to throw out of the window.
- It uses `` - `` as seperator between ``activity.name`` and ``category``. I think this is better looking for the overview. What do you think?

So this helper provides a generic serialization utilizing the standart ``@`` seperator. As far as I can tell current versions of ``hamster-lib`` or ``hamster-gtk`` do not have such a function yet. But share the feeling that we had this around for a while in various incarnations...

Reply:
* Wait, do we intend to not support category-less facts?
* I actually prefer the at sign for consistency or habit.

Reply:
- No we do and want to support category less facts. I am not quite sure that the way legacy hamster did is really where we want to go.. 
- Does that mean you would prefer ``@`` in the overview as well? At this stage I do not care about this detail very much as we will have to revisit the general layout at some later stage anyway...

Reply:
Yes, having `@` in overview is what I am accustomed to.

Reply:
AAlrighty. Will adjust this then :) Im cool either way tbh.